tests/functional/report/checkRoutes.js

```diff
@@ -69,7 +69,7 @@ describe('Report route', () => {
 
     it('should contain config', done => {
         queryReport(done, response => {
-            if (!response.config || !response.config.locationConstraints) {
+            if (!response.config || !response.config.overlayVersion) {
```

Comment:
NOTE: The way we do the report config cleanup has changed: https://github.com/scality/S3/pull/1183/files#diff-2157ec89ef3a5f5eefc407fecbdba988R14